docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -787,7 +787,7 @@ Where to find the hoc templates for the edges:
            "templates": "$COMPONENT_DIR/hoc_templates",
         },
 
-The network is defined by nodes and edges. In the example below, a V1 model is being simulated (with recurrent connections) that receives input from virtual LGN source nodes. Each population of nodes should contain "nodes" and “node_types” while each population of edges should “edges”, “edge_types”.  Gids are assigned to nodes in advance (using another tool) or during the simulation, depending on implementation, and are global across all  populations in the network (see the (gid mapping)[#gid_mapping] section for details)
```

Comment:
This paragraph is a good example where new line per each sentence would simplify reading diff; we can go through formatting issues in a separate change.

Reply:
Yes. In general I do line wrapping, which is already better, but I didn't touch this line. In any case, you should try using --color-words in the command line, it does a much better job than Github no matter how paragraphs are split.

Reply:
There's a button at the top of the diff for "display the rich diff" which I find much easier for looking at the diff.

Reply:
I didn't know that one. For Markdown it really makes it a lot easier to read the differences. However, it doesn't work very well for HTML elements within the Markdown rendering (e.g. the table in which I modified one word in just one cell appears as if it was entirely replaced by a new one).